deploy/ocm-agent-operator-managedfleetnotifications/config.yaml

```diff
@@ -0,0 +1,12 @@
+deploymentMode: "SelectorSyncSet"
+selectorSyncSet:
+  resourceApplyMode: Sync
+  matchExpressions:
+  - key: api.openshift.com/name
+    operator: In
+    values:
+      - "rhobsp02ue1"
```

Comment:
There's nothing specifically _wrong_ with this, but I will just note this is a bit of an anti-pattern from an MCC perspective, we do not normally want to encourage targeting specific clusters (and names are not unique).

It would be fine for this resource to land on the entire fleet, as the 'fleet mode' operation of OCM Agent is not operational on anything but RHOBS clusters.

Reply:
As you already pointed out - I only added this, because it was explicitly called out in the old version of the SOP.
I just removed that selector now.